t/uni/labels.t

```diff
@@ -75,12 +75,17 @@ like $@, qr/Unrecognized character/, "redo to downgradeable labels";
 is $d, 0, "Latin-1 labels are reachable";
 
 {
-    no warnings;
-    goto ここ;
-    
-    if (undef) {
-        ここ: {
-            pass("goto UTF-8 LABEL works.");
+    local $@;
+    eval {
+        goto ここ;
+
+        if (undef) {
+            ここ: {
+               my $x = "jump goto UTF-8 LABEL no longer works";
+            }
         }
-    }
+    };
+    like($@,
+        qr/Use of goto to jump into a construct is no longer permitted/,
+        "Got expected error message");
```

Comment:
From what I can see the purpose of this test is to check whether we can goto a unicode label.

While it still checks that the label is resolved, it no longer checks whether we can goto a unicode label.

Reply:
In b003602b7d, I added a test block which I believe does what you are asking for.  Agreed?

Reply:
@tonycoz ^^